overlord/state/state.go

```diff
@@ -112,6 +119,8 @@ func New(backend Backend) *State {
 		modified:            true,
 		cache:               make(map[interface{}]interface{}),
 		pendingChangeByAttr: make(map[string]func(*Change) bool),
+		taskObservers:       make(map[int]func(t *Task, old Status, new Status)),
+		changeObservers:     make(map[int]func(chg *Change, old Status, new Status)),
```

Comment:
nitpick: should this and other places now use s/obsversers/handelrs/ for consistency with the exported methods?

Reply:
yes definitely good point